src/components/v5/common/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FC, useCallback, useMemo, useState } from 'react';
+import { useIntl } from 'react-intl';
+import clsx from 'clsx';
+import debounce from 'lodash.debounce';
+
+import FormError from '~v5/shared/FormError';
+
+import { TextareaProps } from './types';
+
+const displayName = 'v5.common.Textarea';
+
+const DEFAULT_MAX_CHAR_NUMBER = 90;
+
+const Textarea: FC<TextareaProps> = ({
+  textareaTitle = '',
```

Comment:
Are you sure if we need to initialize these props? I think we don't have to do it. If the prop is optional, we should add a special conditional to display them (as I see, you've done it)